src/hooks/useBottomSheetMaxHeight.ts

```diff
@@ -0,0 +1,39 @@
+import { useWindowDimensions } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
```

Comment:
could we use internal container height instead using `react-native-safe-area-context` library ?